src/SimpleCacheClient.ts

```diff
@@ -16,9 +16,13 @@ export class SimpleCacheClient {
   private readonly controlClient: Momento;
 
   /**
-   * @param {string} authToken
-   * @param {number} defaultTtlSeconds
-   * @param {number} requestTimeoutMs
+   * Create a SimpleCacheClient.
+   * @param {string} authToken - Momento token to authenticate requests with Simple Cache Service.
+   * @param {number} defaultTtlSeconds - A default time to live, in seconds, for cache objects
+   * created by this client.
+   * @param {number} requestTimeoutMs - An optional timeout in milliseconds for get and set operations
```

Comment:
This is probably my fault, but looking at this pr I notice that we are defining optional params in 3 different ways 

- saying its optional in the help text
- `{number=}`
- `{number | undefined}`

while we are updating the docs, maybe we should choose 1 approach and consolidate all the jsdocs. My vote would be for `{number=}` as its the most concise.

Reply:
wdyt?

Reply:
I opted to use the JSDoc style square bracket style  `{number} [requestTimeoutMs]`